.circleci/config.yml.jinja

```diff
@@ -210,6 +210,21 @@ jobs:
                   name: Update branch protection rule from test configuration
                   command: ./scripts/test.py required-status >required_status && cat required_status && ./scripts/update_branch_rule.py required_status
 
+    run-leaderboard:
+        docker:
+            - image: node
```

Comment:
can you pin the tag of this (ex. node:14.4.0) so it's reproducible

Reply:
will do!